components/apimgt/org.wso2.carbon.apimgt.cleanup.service/src/main/java/org/wso2/carbon/apimgt/cleanup/service/organizationPurge/ApplicationPurge.java

```diff
@@ -17,7 +17,116 @@
  */
 package org.wso2.carbon.apimgt.cleanup.service.organizationPurge;
```

Comment:
Package name should be all lower-case. `organizationPurge` needs to be changed see https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html

Reply:
This is not coming from this PR but this needs to be changed. @HiranyaKavishani @mushthaq33 plz note

Reply:
package name should only be  **org.wso2.carbon.apimgt.cleanup.service**